cms/templates/cms/welcome.html

```diff
@@ -1,74 +1,89 @@
-
-        .footnote { color:#ccc; font-size:12px; line-height:18px; }
-
-        @media print, (-o-min-device-pixel-ratio: 5/4), (-webkit-min-device-pixel-ratio: 1.25), (min-resolution: 120dpi) {
-            .logo { background-image:url('{% static "cms/img/logo-white@2x.png" %}'); background-size:181px; }
-        }
-    </style>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>{% trans "django CMS" %}</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
     {% render_block "css" %}
+    <link rel="stylesheet" href="{% static 'cms/css/cms.welcome.css' %}">
 </head>
-<body>
-{% cms_toolbar %}
-<div class="wrapper">
-    <div class="inner">
-        <h1 class="logo">Installation successful!</h1>
-        <p>Switch to <a href="/?{{ request.toolbar.edit_mode_url_on }}">edit mode</a>,<br />log in and start <a href="{% cms_admin_url "cms_page_add" %}?title=Home" id="add-page">adding pages</a>.</p>
-        <p class="buttons">
-            <a href="//docs.django-cms.org/" class="btn" target="_blank">Documentation</a>
-            <a href="//www.django-cms.org/" class="btn" target="_blank">Support</a>
-        </p>
-        <p class="footnote">
-            If you don't see the django CMS logo at the top, make sure you linked the <code>static/cms</code> folder to your static files.</p>
-        <p class="footnote">
-            You're seeing this message because you have <code>DEBUG = True</code> in your
-            django settings file and haven't added any pages yet.
-        </p>
+<body class="cms-welcome-bg">
+    {% cms_toolbar %}
+    <div class="cms-reset cms-welcome">
+        <header class="cms-welcome-header">
+            <h1 class="cms-welcome-heading">
+                <span class="cms-icon cms-icon-check"></span>
+                {% trans "Installation successful!" %}
+            </h1>
+            <div class="cms-welcome-logo">
+                <span class="cms-hidden">{% trans "django CMS" %}</span>
+            </div>
+        </header>
+        {{ request.version }}
+
+        <section class="cms-welcome-section">
+            <h2>
+                {% if user.is_authenticated %}
+                    2. {% trans "Add your first page" %}
+                {% else %}
+                    1. {% trans "Please login" %}
+                {% endif %}
+            </h2>
+            <p>
+                {% blocktrans %}
+                    Welcome to django CMS version <strong>{{ cms_version }}</strong>.
+                {% endblocktrans %}
+                </p>
+            <p>
+                {% url "admin:index" as admin_index %}
+                {% url "admin:cms_page_add" as admin_add_page %}
+                {% blocktrans %}
+                    You can also login through the <a href="{{ admin_index }}">admin</a> and
+                    <a href="{{ admin_add_page }}?title=Home" class="js-welcome-add">add a page</a> manually
```

Comment:
why the "Javascript is disabled" ? if javascript is disabled user is really out of luck, you can't really use cms